.github/workflows/test_conda_store_server_unit.yaml

```diff
@@ -87,13 +87,18 @@ jobs:
       - name: "Install conda-store server 📦"
         run: |
           python -m pip install .
+          python -m pip install pytest-cov
```

Comment:
I think this is best put in `conda-store-server/environment-dev.yaml`, in addition to the dependencies in `conda-store-server/pyproject.toml` under `[tool.hatch.envs.dev]`.